test/static.test.js

```diff
@@ -3703,3 +3703,52 @@ t.test(
     t.end()
   }
 )
+t.test(
+  'converts URL to path',
+  async (t) => {
+    const pluginOptions = {
+      root: url.pathToFileURL(path.join(__dirname, '/static'))
+    }
+    console.log(pluginOptions.root.protocol)
+
+    const fastify = Fastify()
+
+    fastify.register(fastifyStatic, pluginOptions)
+    const response = await fastify.inject({
+      method: 'GET',
+      url: 'foobar.html',
+      headers: {
+        'accept-encoding': '*, *'
+      }
+    })
+    genericResponseChecks(t, response)
+    t.equal(response.statusCode, 200)
+    t.same(response.body, foobarContent)
+    t.end()
```

Comment:
```suggestion
```

Reply:
I guess he oriented himself on the previous tests. But i also agree, we should use t.plan instead of t.end

Reply:
> I guess he oriented himself on the previous tests. But i also agree, we should use t.plan instead of t.end

Done.